### PR TITLE
feat: add storyboard url getter, don't set poster and storyboard urls if the provided tokens aren't valid

### PR DESCRIPTION
**Pull request by gkatsev**

### CODE REVIEW

**Review by cjpillsbury — APPROVED** [review]
Looks awesome.

File: packages/mux-player/src/index.ts [hunk 1]
```diff
@@ -676,6 +678,22 @@ class MuxPlayerElement extends VideoApiElement {
     }
   }
 
+  get storyboard() {
+    if (
+      this.playbackId &&
+      !this.audio &&
+      this.streamType &&
```
Comment (luwes) [inline packages/mux-player/src/index.ts:685]: this should also include an OR with the case that `this.streamType` is not set.

aka this getter should return an URL if `this.streamType` is not set.

**Review by luwes — CHANGES_REQUESTED** [review]
return storyboard url if this.streamType is null

File: packages/mux-player/src/index.ts (hunk 1 above)
Comment (gkatsev) [reply packages/mux-player/src/index.ts:685]: good call 👍🏻 I'll add it in tomorrow.

Comment (gkatsev) [reply packages/mux-player/src/index.ts]: Updated. First, added some tests in https://github.com/muxinc/elements/pull/474/commits/6bd47c8b344f75b569babeb18f9d29e643b9853e to show that it fails and that the fix in https://github.com/muxinc/elements/pull/474/commits/bb0842dc5b0e6fd367b66878a803a84491461d97 would fix the tests

File: packages/mux-player/src/index.ts [hunk 2]
```diff
@@ -678,12 +678,19 @@ class MuxPlayerElement extends VideoApiElement {
...
+   * we aren't an audio player and the stream-type isn't live.
+   */
   get storyboard() {
     if (
       this.playbackId &&
       !this.audio &&
-      this.streamType &&
-      ![StreamTypes.LIVE, StreamTypes.LL_LIVE, StreamTypes.DVR, StreamTypes.LL_DVR].includes(this.streamType as any)
+      (!this.streamType ||
+        (this.streamType &&
```
Comment (luwes) [inline packages/mux-player/src/index.ts:690]: nit (non-blocking): `!this.streamType ||` the right hand of this condition already means `this.streamType` is not null so no need for `(this.streamType && )`

**Review by luwes — APPROVED** [review]
👍

**Pull request merged**